src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -1747,17 +1737,42 @@ public void testInputAnnotationScopeIndentationCheck() throws Exception {
         verifyWarns(checkConfig, fileName, expected);
     }
 
-    @Override
-    protected String internalGetCheckMessage(
-            String messageBundle, String messageKey, Object... arguments) {
-        final ResourceBundle resourceBundle = ResourceBundle.getBundle(
-                messageBundle,
-                Locale.ROOT,
-                Thread.currentThread().getContextClassLoader(),
-                new LocalizedMessage.Utf8Control());
-        final String pattern = resourceBundle.getString(messageKey);
-        final MessageFormat formatter = new MessageFormat(pattern, Locale.ROOT);
-        return formatter.format(arguments);
+    /**
+     * Verifies that the properties of {@link IndentationCheck#MSG_ERROR},
+     * {@link IndentationCheck#MSG_CHILD_ERROR}, {@link IndentationCheck#MSG_CHILD_ERROR_MULTI},
+     * {@link IndentationCheck#MSG_CHILD_ERROR_MULTI} are in numerical order.
+     *
+     * The message properties above are in numerical order currently, on which other tests rely.
+     * If we update the messages in the future, it is required to keep the properties in numerical
+     * order to ensure other tests will work.
+     */
+    @Test
+    public void testPropertyOrderOfErrorMessages() {
```

Comment:
Done. 

The term "property" is taken from @rnveach 's https://github.com/checkstyle/checkstyle/issues/4003#issuecomment-286754290
> ... chop off property 0, and ...

I thought that "property" is a proper noun for "argument" in this context. Maybe I misunderstood his meaning. Sorry for that. My English is not very good. 😅